Mathlib/MeasureTheory/Measure/Dirac.lean

```diff
@@ -179,6 +179,14 @@ instance Measure.dirac.isProbabilityMeasure {x : α} : IsProbabilityMeasure (dir
 instance Measure.dirac.instIsFiniteMeasure {a : α} : IsFiniteMeasure (dirac a) := inferInstance
 instance Measure.dirac.instSigmaFinite {a : α} : SigmaFinite (dirac a) := inferInstance
 
+theorem dirac_eq_one_iff_mem (hs : MeasurableSet s) : dirac a s = 1 ↔ a ∈ s := by
+  rw [Iff.symm (prob_compl_eq_zero_iff hs), ← mem_ae_iff]
```

Comment:
```suggestion
  rw [← prob_compl_eq_zero_iff hs, ← mem_ae_iff]
```
The arrow toward the left reverses things in rewrites, so it's exactly the same as your `Iff.symm`, just more readable.